@xen-orchestra/web-core/lib/components/card/CardTitle.vue

```diff
@@ -0,0 +1,64 @@
+<!-- v1.0 -->
+<template>
+  <div class="card-title">
+    <div class="left-blok">
+      <div class="title-and-counter">
+        <span class="typo h6-medium"><slot name="left"></slot></span>
+        <UiCounter v-if="count" class="count" :value="count" :color="counterColor" />
```

Comment:
According to the design system, the `UiCounter` is used in his `medium` size variant. And by using the size variant, there is no need to use the `count` class to change the font size.
```suggestion
        <UiCounter v-if="count" size="medium" :value="count" :color="counterColor" />
```